example/src/BottomSheet/BottomSheetPage.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { StyleSheet, View } from 'react-native';
+import { BottomSheet, Button, useTheme } from 'smartway-react-native-ui';
+
+export const BottomSheetPage = () => {
+    const theme = useTheme();
+    const [isOpened, setOpened] = useState(false);
+
+    const openBottomSheet = () => {
```

Comment:
should probably use `useCallback()` in order to avoid `unnecessary` rerendering

```typescript
const openBottomSheet = useCallback(() => {
 setOpened(true);
},[])
```

Reply:
nevermind havn't seen it was demo code